src/dotnet/Chat.UI.Blazor/Components/MentionList/MentionList.razor

```diff
@@ -1,36 +1,50 @@
 @namespace ActualChat.Chat.UI.Blazor.Components
+@using ActualChat.Chat.UI.Blazor.Module
 @inherits ComputedStateComponent<MentionSearchResult[]>
 @{
     var mentions = Mentions;
+    var listCls = "";
     if (mentions.Length < 1)
-        return;
+        listCls += "hidden";
     var selection = Selection;
 }
 
-<div class="@Class mb-1 p-1 border border-mention rounded-md overflow-y-auto max-h-96 w-full">
+<div @ref="Ref" class="@Class @listCls mention-list custom-scrollbar">
```

Comment:
Can we apply 'custom-scrollbar' to 'mention-list'?

Reply:
its use in css causes errors